src/lib/Drawer.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import { browser } from '$app/env';
+	import {
+		Button,
+		NavigationDrawer,
+		List,
+		ListItem,
+		Icon,
+		Avatar,
+		Divider
+	} from 'svelte-materialify';
+	import breakpoints, { mediumAndDown, smallAndDown } from './stores/breakpoints';
+	import { drawerVisible } from './stores';
+	import UserInfoDrawer from './UserInfoDrawer.svelte';
+	import { mdiViewDashboard } from '@mdi/js';
+	import { BarChartIcon, UsersIcon } from 'svelte-feather-icons';
+
+	function resize() {
+		breakpoints.update((bps) => bps);
+	}
+
+	$: if (browser) {
+		$mediumAndDown = window.matchMedia($breakpoints['md-and-down']).matches;
+		$smallAndDown = window.matchMedia($breakpoints['sm-and-down']).matches;
+	}
+</script>
+
+<svelte:window on:resize={resize} />
+
+<div class="drawer" class:visible={!$mediumAndDown || $drawerVisible} class:mobile={$mediumAndDown}>
+	<NavigationDrawer class="theme--light">
+		<UserInfoDrawer />
+		<Divider />
+		<List style="padding: 1rem;">
+			<ListItem>
+				<span slot="prepend">
+					<BarChartIcon size="20" />
+				</span>
+				Dashboard
+			</ListItem>
+			<ListItem>
+				<span slot="prepend">
+					<UsersIcon size="20" />
+				</span>
+				Customers
+			</ListItem>
+		</List>
+		<span slot="append" class="pa-2">
+			<Button block>Logout</Button>
+		</span>
+	</NavigationDrawer>
+</div>
+
+<style lang="scss">
```

Comment:
lets not use scss 
lets use css only (you still have css variables)
but if you see the need of scss please point that out